hands_on/local_maxima_part2/test_local_maxima.py

```diff
@@ -23,8 +24,18 @@ def test_find_maxima_empty():
 
 
 def test_find_maxima_plateau():
-    raise Exception('not yet implemented')
+    values = [1, 2, 2, 1]
+    expected_v1 = [1]
+    expected_v2 = [2]
+    expected_v3 = [1, 2]
+
+    assert (np.all(find_maxima(values) == expected_v1) 
+            or np.all(find_maxima(values) == expected_v2) 
+            or np.all(find_maxima(values) == expected_v3))
 
 
 def test_find_maxima_not_a_plateau():
-    raise Exception('not yet implemented')
+    values = np.array([1, 2, 2, 3, 1])
+    expected = np.array([3])
```

Comment:
Strictly speaking, the function takes and returns a list. This works because the array object implements the same functionality of a list that you need for this functions, but in other situation it could fail